altair/api.py

```diff
@@ -233,6 +293,18 @@ def _shape_changed(self, name, old, new):
             self.shape._infer_type(self.parent.data)
 
 
+class Filter():
```

Comment:
The entire Filter class (as it is today) will be just another property inside data?